interfaces/PA-App/src/app/shared/qr-scanner/qr-scanner.component.html

```diff
@@ -47,15 +47,15 @@
 
   <ion-fab vertical="top" horizontal="end" slot="fixed">
     <ion-fab-button *ngIf="torchAvailable"
-                    [title]="'qr-scanner.toggle-flash|translate'"
+                    [title]="toggleFlashTitle"
                     (click)="torchEnabled=!torchEnabled"
                     [color]="torchEnabled ? 'success' : 'light'"
                     class="ion-margin-vertical"
     >
       <ion-icon [name]="torchEnabled ? 'flash' : 'flash-off'"></ion-icon>
     </ion-fab-button>
     <ion-fab-button *ngIf="switchCamerasAvailable"
-                    [title]="'qr-scanner.switch-camera|translate'"
```

Comment:
Ah, I think the type is with the last quote...

```
[title]="'qr-scanner.switch-camera'|translate"
```

I think that would be a better fix.

@jannisvisser

Reply:
go ahead @elwinschmitz ;)